app/src/main/res/layout/activity_login.xml

```diff
@@ -1,20 +1,60 @@
 <?xml version="1.0" encoding="utf-8"?>
-<android.support.constraint.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:app="http://schemas.android.com/apk/res-auto"
+<LinearLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    tools:context="mcs.salazar.jesus.mcsbattleship.LoginActivity">
 
+    tools:context="mcs.salazar.jesus.mcsbattleship.LoginActivity"
```

Comment:
Wait, what? the my name is on the package? lol... Just please erase line above so this "change" doesn't show because it's not really a change

Reply:
lol, I did erase this line of code.